### PR TITLE
Move descheduler to latest upstream

### PR DESCRIPTION
**Pull request by pipo02mix**
Towads https://github.com/giantswarm/giantswarm/issues/31744

### CODE REVIEW

File: .abs/main.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+replace-chart-version-with-git: true
+generate-metadata: true
+chart-dir: ./helm/descheduler
+destination: ./build
```
Comment (fiunchinho) [inline .abs/main.yaml:4]: Please, add a new line at the end of your files. It's happening on several of them.

**Review by fiunchinho — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -3,20 +3,20 @@ orbs:
...
+          # Trigger job on git tag.
         filters:
-            # Trigger the job also on git tag.
           tags:
             only: /^v.*/
           branches:
             ignore:
             - main
-            - master
+            - master
```
Comment (fiunchinho) [inline .circleci/config.yml:22]: You missed this one

**Comment by pipo02mix** [comment]
Thanks, basically I synced with upstream values respecting Network Policy and policy exceptions

File: .circleci/config.yml (hunk 2 above)
Comment (pipo02mix) [inline .circleci/config.yml:22]:
```suggestion
            - master

```

**Review by fiunchinho — APPROVED** [review]

**Comment by fiunchinho** [comment]
do you have any automation or just copying and pasting from upstream?

**Comment by pipo02mix** [comment]
There is some automation described [here](https://handbook.giantswarm.io/docs/product/managed-apps/dev-experience/git-subtree/) but in this project was not configured, and I wanted also to see how all templates are defined in upstream, so I have just copied the files :D

**Pull request merged**